src/containers/Demo/AccordionList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Accordion, Button, Card } from 'react-bootstrap';
+
+interface IAccordionProps {
+  listData: DemoData[];
+}
+
+const AccordionList: React.FC<IAccordionProps> = ({ listData }) => {
+  return (
+    <Accordion>
+      {listData.map(listItem => (
```

Comment:
Move this to a separate function - `renderList`

Reply:
```
return <Accordion>{renderList()}</Accordion>
```

Reply:
Done